from plyer import notification

notification.notify(
    title = "ERROR:エラーが発生しました。",
    message = "不明なエラー",
    app_name = "Deleter",
    timeout = 5
)